Website/src/rendering/src/components/FullPageSearch/FacetList.tsx

```diff
@@ -264,6 +287,7 @@ const FacetList = ({
       </button>
       {activeFilters}
       <div className="facet-list">
+        {isCategoryProductListingPage && <SearchInput onSearchInputChange={onSearchInputChange} />}
```

Comment:
Please move the search input above the active filters.
While being there, could the display condition be moved in a const to keep the JSX condition-free?
```
const searchInput = isCategoryProductListingPage && <SearchInput onSearchInputChange={onSearchInputChange} />;
...
{searchInput}
```